NetKAN/Workshop.netkan

```diff
@@ -34,6 +34,17 @@
                     { "name": "KIS" }
                 ]
             }
+        },
+	{	
+            "version": ">=0.11.0",
+            "override": {
+                "depends": [
+                    { "name": "ModuleManager" },
+                    { "name": "CommunityResourcePack", "minVersion": "0.4.7" },
+		    { "name": "Firespitter Core" },
```

Comment:
oh, sorry that is a typo. is the syntax correct for the minversion of CRP
and to make sure every install of version 0.11 and up use the new
dependencies while older releases use the old dependencies?
Am 17.11.2015 16:15 schrieb "Arne" <notifications@github.com>:

> In NetKAN/Workshop.netkan
> <https://github.com/KSP-CKAN/NetKAN/pull/2615#discussion_r45071054>:
>
> > @@ -34,6 +34,17 @@
> >                      { "name": "KIS" }
> >                  ]
> >              }
> > +        },
> > +	{	
> > +            "version": ">=0.11.0",
> > +            "override": {
> > +                "depends": [
> > +                    { "name": "ModuleManager" },
> > +                    { "name": "CommunityResourcePack", "minVersion": "0.4.7" },
> > +		    { "name": "Firespitter Core" },
>
> The correct id is FirespitterCore, just remove the space
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/KSP-CKAN/NetKAN/pull/2615/files#r45071054>.
>